### PR TITLE
[13.0][ADD] shopfloor dangerous goods

### PR DESCRIPTION
**Pull request by mmequignon**
This module introduces dangerous attributes in shopfloor

Based on:
- https://github.com/OCA/stock-logistics-workflow/pull/854

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -6,3 +6,4 @@ stock-logistics-workflow
 web
 rest-framework
 server-auth
+community-data-files
```
Comment (simahawk) [inline oca_dependencies.txt]: you don't need this. MQT=PIP here

File: shopfloor_dangerous_goods/models/product_product.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    is_lq_product = fields.Boolean(compute="_compute_is_lq_product")
```
Comment (simahawk) [inline shopfloor_dangerous_goods/models/product_product.py]: I don't think this should stay here. All these fields should stay in another module.

File: shopfloor_dangerous_goods/actions/schema.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    _inherit = "shopfloor.schema.action"
+
+    def move_line(self, with_packaging=False, with_picking=False):
+        res = super().move_line(
+            with_packaging=with_packaging, with_picking=with_picking
+        )
+        res["has_lq_products"] = {
+            "type": "boolean",
+            "nullable": False,
+            "required": True,
```
Comment (simahawk) [inline shopfloor_dangerous_goods/actions/schema.py]: I'd say it's NOT required

File: shopfloor_dangerous_goods/tests/test_actions_data.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    def _set_product_lq(cls):
+        limited_amount_lq = cls.env.ref("l10n_eu_product_adr.limited_amount_1")
+        cls.product_a.is_dangerous = True
+        cls.product_a.limited_amount_id = limited_amount_lq
+
+    @classmethod
+    def setUpClass(cls):
+        res = super().setUpClass()
+        cls._set_product_lq()
+        return res
```
Comment (simahawk) [inline shopfloor_dangerous_goods/tests/test_actions_data.py]: return res??? there's no res here

File: shopfloor_dangerous_goods_mobile/static/wms/.gitignore [hunk 5]
```diff
@@ -0,0 +1,21 @@
+.DS_Store
```
Comment (simahawk) [inline shopfloor_dangerous_goods_mobile/static/wms/.gitignore]: this is not needed

File: shopfloor_dangerous_goods/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Shopfloor Dangerous Goods",
+    "summary": "Glue Module Between Shopfloor and l10n_eu_product_adr",
```
Comment (TDu) [inline shopfloor_dangerous_goods/__manifest__.py]:
```suggestion
    "summary": "Glue Module Between Shopfloor and ?",
```

Comment (TDu) [reply shopfloor_dangerous_goods/__manifest__.py]: Is it correct or maybe `stock_dangerous_goods` ?

File: shopfloor_dangerous_goods/tests/test_actions_data.py [hunk 7]
```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 Camptocamp SA
```
Comment (TDu) [inline shopfloor_dangerous_goods/tests/test_actions_data.py:1]: That file should be imported in `__init__.py`

File: shopfloor_dangerous_goods_mobile/README.rst [hunk 8]
```diff
@@ -0,0 +1,79 @@
+================================
```
Comment (TDu) [inline shopfloor_dangerous_goods_mobile/README.rst:1]: I wonder if it would not be better named `shopfloor_mobile_dangerous_goods`

Comment (mmequignon) [reply shopfloor_dangerous_goods_mobile/README.rst:1]: that was my first idea, but @simahawk disagrees

Comment (TDu) [reply shopfloor_dangerous_goods_mobile/README.rst:1]: ok

**Review by TDu — APPROVED** [review]

File: shopfloor_dangerous_goods_mobile/README.rst (hunk 8 above)
Comment (simahawk) [reply shopfloor_dangerous_goods_mobile/README.rst:1]: rationale: keep all related modules under the same "namespace". It's also handy when you list directories or browse modules in your preferred editor ;)

**Review by simahawk — COMMENTED** [review]
LG, can we remove the [WIP]?

**Review by sebalix — APPROVED** [review]

**Review by jgrandguillaume — APPROVED** [review]
LGTM

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
ahhh https://github.com/OCA/stock-logistics-workflow/pull/854 I was convinced it was merge already :sweat:

**Pull request closed without merging**